baseplate/frameworks/queue_consumer/kafka.py

```diff
@@ -419,6 +436,17 @@ class InOrderConsumerFactory(_BaseKafkaQueueConsumerFactory):
     If you need more control, you can create the :py:class:`~confluent_kafka.Consumer`
     yourself and use the constructor directly.
 
+    UPDATE: The InOrderConsumerFactory can NEVER achieve in-order, exactly once message processing.
+
+    Message processing in Kafka to enable exactly once starts at the Producer enabling transactions,
+    and downstream consumers enabling reading exclusively from the committed offsets within a transactions.
+
+    Secondly, without a definied keys consumers consume all partitions at the same time, which could have wildly different order.
+
+    Some improvements are made instead that keeps that "assumed" functionality, but doesn't put pressure on Kafka for committing every single offet.
+
+
+    Instead of committing the offset back to Kafka, commits the offset to it's local offset store, and commits within a certain interval.
```

Comment:
```suggestion
    Instead of committing every single message's offset back to Kafka, the consumer now commits each offset to it's local offset store, and commits the highest seen value for each partition at a defined interval.
```